README.md

```diff
@@ -230,6 +230,34 @@ the `@optional` does not apply, and that value is then checked against the filte
 coercion. These subsequent operations may then cause the result set to be discarded if it does
 not match.
 
+For example, suppose we have two Person nodes with names `Albert` and `Betty` such that there's an edge 
+`Albert-knows->Betty`.
+
+Then for the following query:
+```graphql
+{
+  Person {
+    out_Person_Knows @optional {
+      name @filter(op_name: "=", value: "Charles")
```

Comment:
This is not a legal GraphQL query, this has to be a runtime parameter since literals are not allowed.